frb_example/pure_dart/rust/.flutter_rust_bridge.yml

```diff
@@ -7,4 +7,6 @@ dart_enums_style: true
 c_output: [c_output_path/c_output.h]
 extra_c_output_path:
   - c_output_path_extra/
+llvm-compiler-opts:
+  [-I/usr/lib/clang/15.0.7/include]
```

Comment:
uber-nit: Hmm I am not very sure - shall we hardcode `15.0.7` here? Since other users when using this example may not have this version

Reply:
Sry, I simply forgot to remove it...with setting CPATH it isn't necessary anymore.